docs/configuration/alarms/Discord.md

```diff
@@ -185,28 +191,40 @@ Below is an example of enabling the mini map for pokemon.
 	},
 ```
 
+<!--table-->
 | Parameters     | Description                           | Default     |
 |----------------|---------------------------------------|-------------|
 | `enabled`      | Turns the map on or off               | `true`      |
 | `width`        | Width of the map                      | `250` px    |
 | `height`       | Height of the map                     | `150` px    |
 | `maptype`      | Link to be added to notification text | `roadmap`   |
 | `zoom`         | Specifies the zoom of the map         | `15`        |
+<!--endtable-->
 
 ### Formatting alarms text
 
 Here is a basic guide to apply custom styles to alarm text:
 
+
++------------------------------------+----------------------------------+
```

Comment:
None of this section parses correctly, is this due to markdown differences between GitHub & RTD?

Reply:
For RTD we are using reStructuredText: http://docutils.sourceforge.net/docs/ref/rst/introduction.html

Reply:
Roger